### PR TITLE
Ignore pnpm test for node14

### PR DESCRIPTION
**Pull request by emmansun**
according error message of the actions and https://github.com/pnpm/pnpm/issues/6297

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -167,8 +167,9 @@ module.exports.filesAfter = function (b, n) {
 };
 
 module.exports.shouldSkipPnpm = function () {
-  const REQUIRED_MAJOR_VERSION = 14;
-  const REQUIRED_MINOR_VERSION = 19;
+  // pnpm 8 requires at least Node.js v16.14
+  const REQUIRED_MAJOR_VERSION = 16;
```
Comment (robertsLando) [inline test/utils.js:171]: Shouldn't we check also pnpm version in this check? older pnpm versions also support older node versions

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by emmansun** [comment]
ok, close first.

**Review by robertsLando — APPROVED** [review]

**Comment by justajwolf** [comment]
@baparham this pr is ok，you think？

**Review by baparham — APPROVED** [review]
Looks ok, I approved the workflow to run also, so if that passes this should be good to merge

**Comment by justajwolf** [comment]
@emmansun It is time to merge this pr.

**Comment by baparham** [comment]
> @emmansun It is time to merge this pr.

I believe the controls are setup so that it's on us to do that, but yes, time to merge.

**Pull request merged**